code/modules/clothing/clothing.dm

```diff
@@ -103,7 +103,8 @@
 	var/blood_overlay_type = "suit"
 	var/fire_resist = T0C + 100
 	var/shield_state = "shield-blue"
-
+	
+	COOLDOWN_DECLARE(cooldown_light)
```

Comment:
Why didnt you leave this as a timer cooldown?

Reply:
According to the doc, COOLDOWN_DECLARE is more performant